docs/recipes/let_it_be.md

```diff
@@ -28,7 +28,7 @@ describe BeatleWeightedSearchQuery do
     # ...
   end
 
-  specify { expect(subject.call("joh")).to contain_exactly(@john) }
+  specify { expect(subject.call("john")).to contain_exactly(@john) }
```

Comment:
This is intentional and implies prefix search functionality 🙂 (since this is an example test for a search query object).